services/hapi/src/libs/sync-ratings.js

```diff
@@ -20,66 +19,34 @@ const getUserRatings = async () => {
     reverse: false,
     show_payer: false
   })
```

Comment:
@xavier506 let me know if hard coded variables should be taken from .env variables, right?

  code: 'rateproducer',
  scope: 'rateproducer',

Reply:
yes these should be sourced from env vars as they are different for jungle and mainnet environments